packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -344,36 +320,25 @@ export default class ImageEdit implements EditorPlugin {
      */
     private createWrapper(operation: ImageEditOperation | CompatibleImageEditOperation) {
         //Clone the image and insert the clone in a entity
-        const clone = this.image.cloneNode();
-        const wrappedImage = wrap(clone, KnownCreateElementDataIndex.ImageEditWrapper);
-
-        this.clonedImage = wrappedImage.firstElementChild as HTMLImageElement;
-
-        // Wrap the cloned image with an entity so that we can easily retrieve it later
-        const { wrapper } = insertEntity(
-            this.editor,
-            IMAGE_EDIT_WRAPPER_ENTITY_TYPE,
-            wrappedImage,
-            false /*isBlock*/,
-            true /*isReadonly*/
-        );
-
-        wrapper.id = this.imageId;
-        wrapper.style.maxWidth = '100%';
-        wrapper.style.position = 'fixed';
+        this.clonedImage = this.image.cloneNode(true) as HTMLImageElement;
+        const wrappedImage = wrap(this.clonedImage, KnownCreateElementDataIndex.ImageEditWrapper);
+        this.wrapper = this.editor.getDocument().createElement('span');
+        this.wrapper.appendChild(wrappedImage);
+        this.wrapper.style.maxWidth = '100%';
+        this.wrapper.style.position = 'fixed';
```

Comment:
Can this be added into KnownCreateElementDataIndex.ImageEditWrapper so that one single wrap() call should make all of them working?